text/0000-process-stdio-redirection.md

```diff
@@ -0,0 +1,93 @@
+- Feature Name: process-stdio-redirection
+- Start Date: 2015-04-10
+- RFC PR:
+- Rust Issue:
+
+# Summary
+
+Update the `std::process` API with the ability to redirect stdio of child
+processes to any opened file handle.
+
+# Motivation
+
+The current API in `std::process` allows to either pipe stdio between parent and
+child process or redirect stdio to `/dev/null`. It would also be largely useful
+to allow stdio redirection to any currently opened `std::fs::File` (henceforth
+`File`) handle. This would allow redirecting stdio with a physical file or even
+another process (via OS pipe) without forcing the parent process to buffer the
```

Comment:
When directing to another process, the pipe actually already exist in a sense where `Stdio::piped()` means that the child will be created with a `stdin`/`stdout`/`stderr` handle, under which is just a file descriptor. That may end up being more useful in the long run to hook up processes together perhaps?